index.js

```diff
@@ -96,18 +98,34 @@ function printLinterOutput(res, config, webpack) {
         } else {
           reportOutput = messages;
         }
-        var filePath = loaderUtils.interpolateName(
-          webpack,
-          config.outputReport.filePath,
-          {
-            content: res.results
-              .map(function(r) {
-                return r.source;
-              })
-              .join("\n")
-          }
-        );
-        webpack.emitFile(filePath, reportOutput);
+
+        if (path.isAbsolute(config.outputReport.filePath)) {
+          fs.writeFile(config.outputReport.filePath, reportOutput, function(
```

Comment:
Bad idea use `fs` inside loader, it is break code when you in development mode (we use `memory` fs )

Reply:
@evilebottnawi What do you think about adding outputReport.stream config option to write into custom stream? We can open, for example, file stream in config and write report content into it.

Reply:
What is use case to use absolute?

Reply:
To save report into another folder than `output.path`. I think there's no need to restrict it. In my project I have `reports` folder and I want to save all reports into it. There's no way to do it now.

Reply:
You can use `relative` paths to you folder

Reply:
o_O

Reply:
Why do you think that it's better than to use custom pluggable stream?

Reply:
It is not better, just ask the questions. Anyway we should avoid using `fs` inside loader, so this PR need to rework, feel free to implement other approach

Reply:
How can we do this @evilebottnawi?

Reply:
@ricardogobbosouza hm, look here one solution, use read `fs`, maybe we should add support `fs` in loaders

Reply:
Can we use `fs`?

Reply:
yes, looks no other way to solve problem